RELEASE.md

```diff
@@ -1,421 +0,0 @@
-  - [5.1. RPM Packages](#rpm-packages)
-  - [5.2. DEB Packages](#deb-packages)
-- [6. Build Server](#build-infrastructure)
-- [7. Release Tests](#release-tests)
-- [8. GitHub Release](#github-release)
-- [9. Docker](#docker)
-- [10. Post Release](#post-release)
-  - [10.1. Online Documentation](#online-documentation)
-  - [10.2. Announcement](#announcement)
-  - [10.3. Project Management](#project-management)
-
-## Preparations <a id="preparations"></a>
-
-Specify the release version.
-
-```bash
-VERSION=2.11.0
-```
-
-Add your signing key to your Git configuration file, if not already there.
-
-```
-vim $HOME/.gitconfig
-
-[user]
-        email = michael.friedrich@icinga.com
-        name = Michael Friedrich
-        signingkey = D14A1F16
-```
-
-### Issues <a id="issues"></a>
-
-Check issues at https://github.com/Icinga/icinga2
-
-### Backport Commits <a id="backport-commits"></a>
-
-For minor versions you need to manually backports any and all commits from the
-master branch which should be part of this release.
-
-### Windows Dependencies <a id="windows-dependencies"></a>
-
-In contrast to Linux, the bundled Windows dependencies
-(at least Boost and OpenSSL) aren't updated automatically.
-(Neither by Icinga administrators, nor at package build time.)
-
-To ensure the upcoming Icinga release ships the latest (i.e. most secure) dependencies on Windows:
-
-#### Update packages.icinga.com
-
-Add the latest Boost and OpenSSL versions to
-https://packages.icinga.com/windows/dependencies/ like this:
-
-```
-localhost:~$ ssh aptly.vm.icinga.com
-aptly:~$ sudo -i
-aptly:~# cd /var/www/html/aptly/public/windows/dependencies
-aptly:dependencies# wget https://master.dl.sourceforge.net/project/boost/boost-binaries/1.76.0/boost_1_76_0-msvc-14.2-64.exe
-aptly:dependencies# wget https://master.dl.sourceforge.net/project/boost/boost-binaries/1.76.0/boost_1_76_0-msvc-14.2-32.exe
-aptly:dependencies# wget https://slproweb.com/download/Win64OpenSSL-1_1_1k.exe
-aptly:dependencies# wget https://slproweb.com/download/Win32OpenSSL-1_1_1k.exe
-```
-
-#### Ensure Compatibility
-
-Preferably on a fresh Windows VM (not to accidentally build Icinga
-with old dependency versions) setup a dev environment using the new dependency versions:
-
-1. Download [doc/win-dev.ps1](doc/win-dev.ps1)
-2. Edit your local copy, adjust the dependency versions
-3. Ensure there are 35 GB free space on C:
-4. Run the following in an administrative Powershell:
-   1. `Enable-WindowsOptionalFeature -FeatureName "NetFx3" -Online`
-      (reboot when asked!)
-   2. `powershell -NoProfile -ExecutionPolicy Bypass -File "${Env:USERPROFILE}\Downloads\win-dev.ps1"`
-      (will take some time)
-
-Actually clone and build Icinga using the new dependency versions as described
-[here](https://github.com/Icinga/icinga2/blob/master/doc/21-development.md#tldr).
-Fix incompatibilities if any.
-
-#### Update Build Server, CI/CD and Documentation
-
-* https://git.icinga.com/infra/ansible-windows-build
```

Comment:
And if we also remove this line, we don't actually document internal infra here anymore. Just like you'd like it.